### DISCUSSION TITLE
Difference between Rexie and Rexiebuilder

### DISCUSSION THREAD

**Discussion by afidegnum** [devashishdxt/rexie#18] [Q&A] [answered]
looking at the documentation, I'm a bit confused. Can you please explain the difference between `[Rexie]` and `[Rexiebuilder]` and their use-cases ?

**Comment by devashishdxt** [comment, marked as answer]
`RexieBuilder` is just a [Builder](https://en.wikipedia.org/wiki/Builder_pattern) for `Rexie`. You can get an instance of `RexieBuilder` from `Rexie::builder()` and after configuration, obtain the final `Rexie` instance using `RexieBuilder::build()`.

**Reply by afidegnum** [reply]
Thanks,
